.github/workflows/pr-author-assigner.yml

```
@@ -0,0 +1,12 @@
+name: Auto assign author to PR
+on:
+    pull_request:
+        types: [ opened ]
+
+jobs:
+    assignAuthor:
+        runs-on: ubuntu-latest
+        steps:
```

I like the fix, but can we add this tiny condition for the auto-assign action?

```suggestion
        if: github.event.pull_request.draft == false
        steps:
```

---

makes sense =D applied change